agent-inject/handler.go

```diff
@@ -142,11 +151,20 @@ func (h *Handler) Handle(w http.ResponseWriter, r *http.Request) {
 		msg := fmt.Sprintf("error marshalling admission response: %s", err)
 		http.Error(w, msg, http.StatusInternalServerError)
 		h.Log.Error("error on request", "Error", msg, "Code", http.StatusInternalServerError)
+		incrementInjectionFailures(admReq.Request.Namespace)
```

Comment:
We may also want to call `incrementInjectionFailures()` for the error returns further up in this function too? But this should suffice as-is; I could even see those being a different error metric added in the future.